articles/purview/catalog-permissions.md

```diff
@@ -71,8 +71,8 @@ For users who have access to their company's [Azure portal](https://portal.azure
 |I need to enable a Service Principal or other programmatic identity to set up and monitor scans in Azure Purview without allowing the programmatic identity to access the catalog's information |Purview Data Source Administrator Role|
 |I need to put users into roles in Azure Purview | Owner or User Access Administrator |
 
-Advance to the next article to learn how to add a security principal to a role.
+For more information on how to add a security principal to a role, refer [Quickstart: Create an Azure Purview account](create-catalog-portal.md) .
```

Comment:
(sp) `refer` -> `refer to`

Reply:
@helizabethwhite - Thanks for the review. I have updated the sentence as suggested, please take a look and suggest your thoughts.